src/main/java/org/cotato/csquiz/domain/recruitment/service/RecruitmentNotificationService.java

```diff
@@ -30,4 +42,23 @@ public void requestRecruitmentNotification(String recruitEmail, boolean isPolicy
                 RecruitmentNotificationRequester.of(recruitEmail, isPolicyChecked)
         );
     }
+
+    @Transactional(readOnly = true)
+    public RecruitmentNotificationLogsResponse findNotificationLogs() {
+        List<RecruitmentNotification> top5Notification = recruitmentNotificationReader.findTopNLatestNotifications(5);
+
+        List<Long> top5NotificationIds = top5Notification.stream()
+                .map(RecruitmentNotification::getId)
+                .toList();
+        HashMap<Long, List<RecruitmentNotificationEmailLog>> logsByNotificationId = recruitmentNotificationEmailLogReader.groupByNotificationIds(
+                top5NotificationIds);
```

Comment:
Reader 계층 추가의 이점(?)을 얻기 위해서 요런식으로 수정하면 어떨까요? 

Id를 찾아서 조회하는 세부 구현은 구현 레이어로 숨기는거죠

```suggestion
        HashMap<Long, List<RecruitmentNotificationEmailLog>> logsByNotificationId = recruitmentNotificationEmailLogReader.groupByNotification(top5Notification);
```

Reply:
[6267035](https://github.com/IT-Cotato/COTATO-BE/pull/357/commits/6267035197246d28e11e55c21297c59e1395c781)

Reply:
[0b5457b](https://github.com/IT-Cotato/COTATO-BE/pull/357/commits/0b5457bb005fe946528baa12c9a4fbedf00bea91)